internal/ethapi/backend.go

```diff
@@ -86,6 +86,8 @@ type Backend interface {
 
 	ChainConfig() *params.ChainConfig
 	Engine() consensus.Engine
+	GetSequencerRPCService() *rpc.Client
+	GetGenesis() *types.Block
```

Comment:
Nit: in Go we only prefix getters with `Get` if there's some collision with a field name on a struct.

Reply:
SGTM, made that change.